### DISCUSSION TITLE
Import nether / end worlds

### DISCUSSION THREAD

**Discussion by MrPengujn** [itzg/docker-minecraft-server#1841] [Q&A]
Is there a way to import nether / end instead of default generating them?

**Comment by itzg** [comment]
You can always supply existing world data using https://github.com/itzg/docker-minecraft-server#cloning-world-from-a-container-path

**Reply by MrPengujn** [reply]
I'm talking about importing all three at once [ overworld, nether, end ], I'm migrating an old server

**Reply by MrPengujn** [reply]
so, I kept trying to make it work but when copying file directly java doesn't have perms to modify world files

**Reply by itzg** [reply]
> I'm talking about importing all three at once [ overworld, nether, end ], I'm migrating an old server

That's what I was addressing 😕. What part of that section didn't seem correct?

> java doesn't have perms to modify world files

The container runs the server as UID 1000. So make sure the files are owned by that UID.

**Reply by MrPengujn** [reply]
I must be having 0 braincells
even changing the owner and groups doesn't work
that section isn't helping much

Forgot to specify that I'm trying to import data from a spigot server which means that I have three separate world files, nothing works, quite literally

**Reply by itzg** [reply]
> I must be having 0 braincells
> 
> even changing the owner and groups doesn't work

That's all it comes down to. It's just regular Linux file permissions/ ownership going on. Not sure what else to suggest.

> that section isn't helping much

Can't explain it any clearer than the docs and diagram I put there

> Forgot to specify that I'm trying to import data from a spigot server which means that I have three separate world files, 

That's handled fine by the section I mentioned. Hopefully you set https://github.com/itzg/docker-minecraft-server#running-a-bukkitspigot-server
